src/plugins/fs/generic.c

```diff
@@ -204,6 +199,35 @@ gboolean bd_fs_wipe (const gchar *device, gboolean all, GError **error) {
     return TRUE;
 }
 
+/**
+ * bd_fs_wipe:
+ * @device: the device to wipe signatures from
+ * @all: whether to wipe all (%TRUE) signatures or just the first (%FALSE) one
+ * @error: (out): place to store error (if any)
+ *
+ * Returns: whether signatures were successfully wiped on @device or not
+ *
+ * Tech category: %BD_FS_TECH_GENERIC-%BD_FS_TECH_MODE_WIPE
+ */
+gboolean bd_fs_wipe (const gchar *device, gboolean all, GError **error) {
+    return _fs_wipe (device, all, FALSE, error);
```

Comment:
`force=FALSE` **is** a change in behavior here, right? AFAIK, #483 mentions we **don't want** to change behavior.

Reply:
Oops, fixed. I even added test to check the behaviour changed :man_facepalming:

Reply:
Yeah, that was a bit confusing :smile: